plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1295,8 +1295,12 @@ blockly.block.feature_simple_random_selector=Random feature from list:
 blockly.block.feature_simple_random_selector.input=feature:
 blockly.block.feature_simple_random_selector.empty=List cannot be empty
 blockly.block.feature_tree.decorator_input=decorator:
+blockly.block.feature_tree_pine=Tree of type %1 with dirt: %2 trunk: %3 foliage: %4 foliage height: %5 roots: %6 height of %7 + at most %8 + at most %9 %10 always place dirt: %11 %12 ignore vines: %13 %14 decorators:
+blockly.block.feature_tree_pine.empty=(None)
 blockly.block.feature_tree_simple=Tree of type %1 with dirt: %2 trunk: %3 foliage: %4 roots: %5 height of %6 + at most %7 + at most %8 %9 always place dirt: %10 %11 ignore vines: %12 %13 decorators:
 blockly.block.feature_tree_simple.empty=(None)
+blockly.block.feature_tree_spruce=Spruce tree with dirt: %1 trunk: %2 foliage: %3 max foliage radius: %4 exposed trunk blocks: %5 roots: %6 height of %7 + at most %8 + at most %9 %10 always place dirt: %11 %12 ignore vines: %13 %14 decorators:
```

Comment:
I would call that "foliage radius" only, given it specifies radius, not its maximal value

Reply:
I went with "max foliage radius" because the leaves might not reach that radius if e.g. the tree is too short

Reply:
Ok, makes sense